test_tipc/test_train_inference_python_npu.sh

```diff
@@ -49,8 +49,8 @@ grep -n '.yml' $FILENAME  | cut -d ":" -f 1 \
 | while read line_num ; do 
     train_cmd=$(func_parser_value "${lines[line_num-1]}")
     trainer_config=$(func_parser_config ${train_cmd})
-    echo ${trainer_config}
     sed -i 's/use_gpu/use_npu/g' "$REPO_ROOT_PATH/$trainer_config"
+    sed -i 's/aligned: True/aligned: False/g' "$REPO_ROOT_PATH/$trainer_config"
```

Comment:
change aligned to false when running on npu; RoiAlign npu kernel don't support aligned=true.